include/phasar/PhasarLLVM/TypeHierarchy/VFTable.h

```diff
@@ -21,19 +21,19 @@ template <typename F> class VFTable {
 public:
   virtual ~VFTable() = default;
 
-  virtual F getFunction(unsigned Idx) const = 0;
+  [[nodiscard]] virtual F getFunction(unsigned Idx) const = 0;
 
-  virtual std::vector<F> getAllFunctions() const = 0;
+  [[nodiscard]] virtual std::vector<F> getAllFunctions() const = 0;
 
-  virtual int getIndex(F f) const = 0;
+  virtual int getIndex(F F) const = 0;
```

Comment:
Please rename variable `F` to `Fun` as it will otherwise collide with the name of the template parameter.